README.md

```diff
@@ -44,6 +44,40 @@ can be used as a reference for setup:
 * [Standard deployment](./kube/prometheus-meta.yaml)
 * [Deployment with the Prometheus Operator](./kube/prometheus-meta-operated.yaml)
 
+### Configuration
+
+The majority of configuration options for the sidecar are set through flags. To see all available flags, run:
+
+```
+stackdriver-prometheus-sidecar --help
+```
+
+#### Filters
+
+The `--filter` flag allows to provide filters which all series have to pass before being sent to Stackdriver. The flag may be repeated to provide several filters. The filter syntax is based on the well-known PromQL label matchers, e.g.:
```

Comment:
"is based on" is vague; how is it different? Would it be fair to say "is the same as"?

Reply:
Yes, makes sense.